cloudflare/resource_cloudflare_load_balancer_monitor.go

```diff
@@ -93,7 +93,7 @@ func resourceCloudflareLoadBalancerMonitor() *schema.Resource {
 
 			"expected_body": {
 				Type:     schema.TypeString,
-				Optional: true,
+				Required: true,
```

Comment:
this will raise an error when creating a tcp monitor because 'expected_body' is only valid with http(s) monitors